### ISSUE TITLE
General evaluation - it§s a pity that some features are missing

### ISSUE THREAD

**Issue by lubosdz** [yeesoft/yii2-yee-cms#55]
Hi, 

just want to share briefly my experience after evaluating few CMS based on Yii2 (Luya, CraftCMS, YeeCms, EasyiiCms, SkeeksCms).

YeeCms has purest design and is well designed. It is not overengineered, therefore performant and easy to get familiar with. It follows partially WordPress administrative UI - e.g. supports comments, uploading images/files, gallery, TinyMC wisywig, basic SEO and has multilanguage support. Things which lack other CMSs or are paid / licensed.

However there are some drawback which prevent it from easy usage:

* few SQL crashes, e.g. when creating category: 

```
SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'title' in where clause is ambiguous
The SQL being executed was: SELECT COUNT(*) FROM `media_category` LEFT JOIN `media_category_lang` ON `media_category`.`id` = `media_category_lang`.`media_category_id` WHERE (`slug` LIKE '%testcat0001%') AND (`title` LIKE '%testcat0001%')
```

* creating page misses some crucial features:

   * setting **parent** (like in WP) - the most important feature.
   * page redirects
   * hide from navigation menu
   * page order in hierarchy
   * featured image or intro text would also add value
   * not well documented ... how to create pages under categories, theme applying, localization & translation ...

It's a pity, that these few things degrade overally very promising application.

**Comment by arollmann** [comment]
Indeed, this project seems to have become dormant. Regarding the CMS built around Yii framework, I strongly recommend looking to LUYA, which is much more advanced and more modern in its design.

**Comment by lubosdz** [comment]
Rather than Luya I would recommend CraftCMS. Luya has some issues like non-indexed tables (after 2+ yrs of development ?!), using heavily angular frontend which increases learning curve (also makes fixing and custom development costy) and personally I really did not like the backend - not intuitive, some things buggy ... CraftCMS has more consistent development roadmap, standard libraries, releases almost every week and is truly application written `by developers for developers`. It's only a pity, that it is based on licensing business model, but in general it's worth of those 200 USD considering support and feature richness. 

We needed something in between - lightweight CMS (minimum external dependencies) with good performance and great extensibility - Yee was our hope. But the only way would be to re-start development from scratch ...
